### PR TITLE
Updated Upgrade Documentation

### PR DESCRIPTION
**Pull request by rishabh-dell**
# Description
Updated Document for Driver Upgrade using Operators

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/648 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/operator/_index.md [hunk 1]
```diff
@@ -256,7 +256,18 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
     $ kubectl edit csiunity/unity -n <driver-namespace>
     ```
     and modify the installation. The usual fields to edit are the version of drivers and sidecars and the env variables.
-* Modify the API object in place via `kubectl patch` command. 
+    To get the name of the installed driver, which is `unity` in the above case, run the below command.
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/operator/_index.md:259]: Replace "the below" with "this"

**Review by satyakonduri — APPROVED** [review]

File: content/docs/csidriver/installation/operator/_index.md (hunk 1 above)
Comment (rishabh-dell) [reply content/docs/csidriver/installation/operator/_index.md]: acknowledged

**Review by AkshaySainiDell — APPROVED** [review]

**Review by AkshaySainiDell — APPROVED** [review]

File: content/docs/csidriver/installation/operator/_index.md [hunk 2]
```diff
@@ -250,13 +250,35 @@ Please refer to the _Troubleshooting_ section [here](../../troubleshooting/opera
...
 
 * Modifying the installation directly via `kubectl edit`
-    For example - If the name of the installed Unity XT driver is unity, then run
+    For example - If the Unity XT driver is installed then run this command to get the name of the driver.
     ```
     # Replace driver-namespace with the namespace where the Unity XT driver is installed
-    $ kubectl edit csiunity/unity -n <driver-namespace>
+    $ kubectl get csiunity -n <driver-namespace>
+    ```
+    use the driver name in `kubectl edit` command.
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md:258]: It is not the driver name. Please change it to "Object name of kind CSIUnity"

File: content/docs/csidriver/installation/operator/_index.md [hunk 3]
```diff
@@ -250,13 +250,35 @@ Please refer to the _Troubleshooting_ section [here](../../troubleshooting/opera
...
+
+    ```
+    # Replace driver-name with the name of the driver
+    $ kubectl edit csiunity/<driver-name> -n <driver-namespace>
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md:262]: Same comment as above

File: content/docs/csidriver/installation/operator/_index.md [hunk 4]
```diff
@@ -250,13 +250,35 @@ Please refer to the _Troubleshooting_ section [here](../../troubleshooting/opera
...
+    For example - If the Unity XT driver is installed then run this command to get the name of the driver.
     ```
     # Replace driver-namespace with the namespace where the Unity XT driver is installed
-    $ kubectl edit csiunity/unity -n <driver-namespace>
+    $ kubectl get csiunity -n <driver-namespace>
+    ```
+    use the driver name in `kubectl edit` command.
+
+    ```
+    # Replace driver-name with the name of the driver
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md:261]: Same comment as above

**Review by rensyct — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Pull request merged**